nuxeo-core/nuxeo-core-api/src/main/java/org/nuxeo/ecm/core/api/model/impl/AbstractProperty.java

```diff
@@ -472,8 +472,8 @@ protected Serializable getValueDeprecation() {
                     msg.append("Return value from deprecated property");
                 } else {
                     msg.append("Return value from '")
-                       .append(deprecatedFallback.getXPath())
-                       .append("' if not null, from deprecated property otherwise");
+                    .append(deprecatedFallback.getXPath())
```

Comment:
Indentation was correct here, can you check the formatter and rollback this change?

Reply:
done